skimage/feature/__init__.py

```diff
@@ -27,6 +28,17 @@
 from .haar import (haar_like_feature, haar_like_feature_coord,
                    draw_haar_like_feature)
 
+from ..registration.register_translation import (
+    register_translation as _register_translation)
```

Comment:
This makes my head spin. Can you rename the file something like `_fourier_space_translation` or somesuch? In general, we have tried to avoid files (modules) having the exact same name as functions because then it is ambiguous when you write `registration.register_translation` whether you mean the function or the module.

Reply:
(Note: I realise that the original file doesn't follow this convention. We haven't been great at enforcing this policy but I think it's worth enforcing going forward.)